runtime/lua/vim/lsp/protocol.lua

```diff
@@ -427,6 +427,12 @@ function protocol.make_client_capabilities()
           properties = { 'edit' },
         },
       },
+      codeLens = {
+        dynamicRegistration = false,
+        resolveSupport = {
+          properties = { 'command', 'title' },
```

Comment:
Can the title be resolved lazily? If I understand [the code](https://github.com/neovim/neovim/blob/0a0962a2e8d9ffa5f03e492fa293e964650a965e/runtime/lua/vim/lsp/codelens.lua#L234) correctly only the command is resolved.

Reply:
Good point. The title is actually part of the command